carto_report/report.py

```diff
@@ -24,614 +25,595 @@
+                            <li class="as-list__item as-color--complementary">Quota Left: {{left_storage}} MB, {{pc_left}} %</li>
+                        </ul>
+                    </div>
+                    <div class="as-box">
+                        <h2 class="as-title">
+                            Location Data Services
+                        </h2>
+                        <div class="as-box" id="lds-table">
+                            {{lds.to_html()}}
+                        </div>
+                        <div class="as-box" id="lds-fig">
+                            {{html_fig_lds}}
+                        </div>
+                    </div>
+                </main>
+                <aside class="as-sidebar as-sidebar--right">
+                <div class="as-container">
+                    <div class="as-box as-title as-font--medium">
+                    Datasets
+                    </div>
+                    <div class="as-box">
+                        <h2 class="as-title">
+                            Datasets Summary
+                        </h2>
+                        <ul class="as-list">
+                            <li class="as-list__item as-font--medium">Number of tables: {{total_dsets}}</li>
+                            <li class="as-list__item">Sync tables: {{sync}}</li>
+                            <li class="as-list__item">Tables Size: {{total_size_tbls}} MB</li>
+                        </ul>
+                    </div>
+                    <div class="as-box">
+                    <h2 class="as-title">
+                        Privacy
+                    </h2>
+                    <ul class="as-list">
+                        <li class="as-list__item as-color--support-01">🔒 Private: {{private}} tables</li>
+                        <li class="as-list__item as-color--support-02">🔗 Shared with link: {{link}} tables</li>
+                        <li class="as-list__item as-color--support-03">🔓 Public: {{public}} tables</li>
+                    </ul>
+                    </div>
+                    <div class="as-box">
+                    <h2 class="as-title">
+                        Geometry
+                    </h2>
+                    <p class="as-body">
+                        Number of geocoded tables: {{geo}}
+                    </p>
+                    <ul class="as-list">
+                        <li class="as-list__item">📌 Points: {{points}} tables</li>
+                        <li class="as-list__item">〰️ Lines: {{lines}} tables</span></li>
+                        <li class="as-list__item">⬛ Polygons: {{polys}} tables</li>
+                    </ul>
+                    <p class="as-body">
+                        Number of non-geocoded tables: {{none_tbls}}
+                    </p>
+                    </div>
+                    <div class="as-box" id="tables-size">
```

Comment:
These two tables lack a proper title to understand what they are referring to, they look like two lists of five random datasets

Reply:
I will add the required titles.

Reply:
ok, please add it as a new issue, or check on your pending tasks